lerna.json

```diff
@@ -0,0 +1,15 @@
+{
+    "packages": [
+        "packages/*"
+    ],
+    "version": "5.0.0",
+    "command": {
+        "bootstrap": {
+            "hoist": true
+        },
+        "publish": {
```

Comment:
Should we set [`changelogPreset`](https://github.com/lerna/lerna/tree/master/commands/version#--create-release-type) to `conventionalcommits`?

Reply:
The reason for using conventionalcommits is that it works with breaking changes in the commit title, like: `feat!: xxx`, while the default doesn’t (angular).

On the other hand, we get a little bit uglier changelogs, but I prefer correctness.

Reply:
Note: please consider applying all these small things in our eslint config.

Reply:
Done ✅

Reply:
PR for ESLint config: https://github.com/moxystudio/eslint-config/pull/100